nuxeo-distribution/nuxeo-functional-tests/src/main/java/org/nuxeo/functionaltests/shibboleth/ShibbolethTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * (C) Copyright 2017 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Funsho David
+ *
+ */
+
+package org.nuxeo.functionaltests.shibboleth;
+
+import org.nuxeo.functionaltests.AbstractTest;
+import org.nuxeo.functionaltests.pages.shibboleth.ShibbolethLoginPage;
+
+/**
+ * @since 9.10
+ */
+public class ShibbolethTest extends AbstractTest {
+
+    protected ShibbolethLoginPage loginAsShibbolethUser(String url, String username, String password) {
```

Comment:
I wonder what is the point of returning a `ShibbolethLoginPage`: once logged in, aren't you redirected to another kind of page?

Reply:
I agree and I need to dig further into this later, but not returning this page made the tests get stuck once logged in. I'll see for reworking it when the tests will be activated.